### PR TITLE
Alma Linux 8.7 and Ubuntu 20.04 images for compute nodes

### PR DESCRIPTION
**Pull request by edwardsp**
Deploying successfully

### CODE REVIEW

File: config.tpl.yml [hunk 1]
```diff
@@ -264,6 +264,13 @@ images:
   #   os_type: Linux # Linux or Windows
   #   version: 7.9 # Version of the image to create the image definition in SIG. Pattern is major.minor where minor is mandatory
 # Pre-defined images
+  - name: azhop-almalinux86-v2-rdma-gpgpu
```
Comment (xpillons) [inline config.tpl.yml:267]: This name doesn't map to any packer ones if customer wants to use it. The provided one is to build marketplace image, not custom images. Change the name to the packer script file name.

**Review by xpillons — CHANGES_REQUESTED** [review]

File: packer/scripts/almalinux/hpc_image.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+set -e
+yum install -y git
+
+df -h
+
+mkdir -p /mnt/resource
+chmod 777 /mnt/resource
+cd /mnt/resource
+
+git clone https://github.com/edwardsp/azhpc-images # -b centos-hpc-20220112
```
Comment (xpillons) [inline packer/scripts/almalinux/hpc_image.sh:11]: can we avoid this dependency and uses the regular public repo on a fix version and apply a patch ?

File: playbooks/roles/cyclecloud_cluster/projects/common/cluster-init/files/nhc/nhc_common.conf.j2 [hunk 3]
```diff
@@ -56,7 +56,7 @@
 # Not present on Alma and Ubuntu
 # * || check_ps_service -S -u root crond
 # * || check_ps_service -r -d qmgr postfix
- * || check_ps_service -S -u root sshd
+# * || check_ps_service -S -u root sshd
```
Comment (vgamayunov) [inline playbooks/roles/cyclecloud_cluster/projects/common/cluster-init/files/nhc/nhc_common.conf.j2:59]: sshd check fails on ubuntu-20.04 due to different output of 'ps'

**Comment by vgamayunov** [comment]
- Created Alma Linux 8.7 from base image
- Created Ubuntu 20.04 image from ubuntu-hpc (dsvm)
- Lustre client from AML
- No support for Alma Linux 8.6

File: config.tpl.yml [hunk 4]
```diff
@@ -267,6 +267,13 @@ images:
   #   os_type: Linux # Linux or Windows
   #   version: 7.9 # Version of the image to create the image definition in SIG. Pattern is major.minor where minor is mandatory
 # Pre-defined images
+  - name: azhop-compute-almalinux-8_6
```
Comment (xpillons) [inline config.tpl.yml:270]: This need to be changed to 8_7

File: config.tpl.yml [hunk 5]
```diff
@@ -267,6 +267,13 @@ images:
...
+    publisher: azhop
+    offer: almalinux
+    sku: 8_6-hpc-gen2
+    hyper_v: V2
+    os_type: Linux
+    version: 8.6
   - name: azhop-almalinux85-v2-rdma-gpgpu
```
Comment (xpillons) [inline config.tpl.yml:277]: alma 8.5 should be removed

File: packer/scripts/centos/slurm.sh [hunk 6]
```diff
@@ -86,12 +86,9 @@ fi
 # Install Enroot
 #
 
-ENROOT_VERSION_FULL=${1:-3.4.0-2}
-ENROOT_VERSION=${ENROOT_VERSION_FULL%-*}
-
 arch=$(uname -m)
-rpm -q enroot || yum install -y https://github.com/NVIDIA/enroot/releases/download/v${ENROOT_VERSION}/enroot-${ENROOT_VERSION_FULL}.el7.${arch}.rpm
-rpm -q enroot+caps || yum install -y https://github.com/NVIDIA/enroot/releases/download/v${ENROOT_VERSION}/enroot+caps-${ENROOT_VERSION_FULL}.el7.${arch}.rpm
+yum install -y https://github.com/NVIDIA/enroot/releases/download/v3.4.1/enroot-3.4.1-1.el8.${arch}.rpm
```
Comment (xpillons) [inline packer/scripts/centos/slurm.sh:90]: do we expect to install el8 RPMS on centos 7.9 ?

**Review by xpillons — CHANGES_REQUESTED** [review]

File: packer/scripts/almalinux/hpc_image.sh (hunk 2 above)
Comment (edwardsp) [reply packer/scripts/almalinux/hpc_image.sh:11]: not easily at the moment - an issue has been created to resolve - #1395

**Review by xpillons — APPROVED** [review]

**Pull request merged**